[New Product] Red Hat JBoss Enterprise Application Platform

adriens opened the pull request:
cf : 

- https://github.com/endoflife-date/endoflife.date/issues/2882

adriens committed:
Create redhat-jboss-eap.md

Working on https://github.com/endoflife-date/endoflife.date/issues/2882

adriens committed:
Update redhat-jboss-eap.md

cf https://github.com/endoflife-date/endoflife.date/issues/2882

adriens committed:
Update redhat-jboss-eap.md

cf https://github.com/endoflife-date/endoflife.date/issues/2882

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+> Red Hat JBoss Enterprise Application Platform (JBoss EAP) is an open-source Java EE (Enterprise Edition) application server.
+> It is a commercially supported and certified version of the open-source WildFly application server.
+> JBoss EAP provides a platform for developing and deploying enterprise-level Java applications.
+
```
The two quote blocks must be merged (see https://deploy-preview-2883--endoflife-date.netlify.app/redhat-jboss-eap).

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+activeSupportColumn: Full Support
+eolColumn: Maintenance Support
+releaseDateColumn: true
+extendedSupportColumn: Extended Life Cycle Support
```
There are ELS 1 and ELS 2 support and `extendedSupport` dates are those of ELS 2. Should this label changed to `Extended Life Cycle Support (2)` ?

adriens replied:
Yes, I was not sure we should but bey seing your remark, it makes me feel this is the right way. Still I don't know which tags to add : could you just add the proper one on a single release... and i'll fill the rest of them ? :pray:

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+    releaseDate: 2016-05-01
+    lts: 2026-11-30
+    latestReleaseDate: 2023-03-01
+    link: https://access.redhat.com/articles/6716841
```
This page mention the latest update on 2022-11-15, but `latestReleaseDate` is 2023-03-01. Is there a source we could go to check `latestReleaseDate`s ?

adriens replied:
cf https://access.redhat.com/articles/6999113 
![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/11c846a9-5cad-4f9b-a21d-8814f86aea99)

adriens replied:
Also https://access.redhat.com/articles/2332721

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+-   releaseCycle: "6"
+    support: 2016-06-30
+    eol: 2019-06-30
+    extendedSupport: 2025-06-30
```
Should be December 30, 2028 ?

adriens replied:
Working on it

adriens replied:
https://access.redhat.com/articles/3751821

![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/91e7058f-cf76-42c4-a4e2-1a6168f6f991)

adriens replied:
According to this : 

- End of ELS-1 : 2022-06
- End of ELS-2 : 2025-06

Which one should we keep ?

marcwrobel replied:
According to https://access.redhat.com/support/policy/updates/jboss_notes#phase_compare no security or bug fixes are issued in ELS 2, so I would say ELS-1. This is also something that must be documented in the product description.

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+    support: 2016-06-30
+    eol: 2019-06-30
+    extendedSupport: 2025-06-30
+    latest: "6.4"
```
6.4.10 according the the link below (https://access.redhat.com/articles/3099701) ?

adriens replied:
https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/6.4

[`6.4.24`](https://access.redhat.com/articles/6958003)

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+    latest: "6.4"
+    releaseDate: 2012-06-01
+    lts: 2022-06-30
+    latestReleaseDate: 2022-06-30
```
Where can I double-check this date (it does not seem in line with https://access.redhat.com/articles/3099701) ?

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+
+-   releaseCycle: "5"
+    support: 2013-11-30
+    eol: 2013-11-30
```
2016 ?

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+-   releaseCycle: "5"
+    support: 2013-11-30
+    eol: 2013-11-30
+    extendedSupport: 2019-11-30
```
This is the ELS 1 date, should be 2022 for ELS 2.

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+    extendedSupport: 2029-11-30
+    latest: "7.4.6"
+    releaseDate: 2016-05-01
+    lts: 2026-11-30
```
I do not understand the notion of LTS for JBoss EAP : all versions are LTS and it is not defined in the description. I think this field should be removed.

adriens replied:
Done :+1:

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+    support: 2013-11-30
+    eol: 2013-11-30
+    extendedSupport: 2019-11-30
+    latest: "5.2"
```
5.2.0 according to the link below (https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html/release_notes_5.2.0/index).

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+    latest: "5.2"
+    releaseDate: 2009-11-01
+    lts: 2019-11-30
+    latestReleaseDate: 2013-12-23
```
Where can I double-check this date ?

adriens replied:
Working on it

adriens replied:
https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html/release_notes_5.2.0/index

adriens replied:
https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/5

adriens replied:
https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html/release_notes_5.2.0/appe-release_notes_5.2-revision_history

adriens replied:
https://access.redhat.com/support/policy/updates/jboss_notes/
![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/0999efcf-aa70-4143-a3cc-3ca76c7091af)

adriens replied:
Could not find better than this : 
[2013-10-31](https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html-single/release_notes_5.2.0/index#appe-Release_Notes_5.2-Revision_History)

![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/ed0e25a7-b6e7-4429-a840-4f58d6821875)

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+    latestReleaseDate: 2013-12-23
+    link: https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html/release_notes_5.2.0/index
+
+-   releaseCycle: "4"
```
According to https://access.redhat.com/support/policy/updates/jboss_notes#Life_cycle_dates, 4.3 and 4.2 were two distinct cycles with their own support dates. I think we should split those versions in this page too.

adriens replied:
Working on it :+1:

adriens replied:
Hard to find other 4.2.x versions ;-(

marcwrobel commented on products/redhat-jboss-eap.md:
```diff
@@ -0,0 +1,153 @@
+
+## Full Support Phase
+
+Start Date: General Availability
```
The description is a copy / paste of https://access.redhat.com/support/policy/updates/jboss_notes#phases. This description should summarize the EOL / support policy, users can always look at the `releasePolicyLink` if they need details. Moreover some parts here cannot be understood by the users if they are not explained (or linked to other document like it is done on https://access.redhat.com/support/policy/updates/jboss_notes#phases).

Some example of things I found cryptic or unnecessary :

- _...to the published Scope of Coverage and Service Level Agreement_ : What are those ? Where can I find those documents ?
- _Container distributions of Application Services products in the ELS Life Cycle Phase will be marked "Deprecated" in the Red Hat Container Catalog._ : Is it really useful on endoflife.date ?
- _The Limited Product Life Cycle of three years is generally applied to newer product areas and rapidly changing
technologies such as development frameworks._ : What does it mean for JBoss EAP users ?

adriens replied:
Working on it :+1:

adriens committed:
Update redhat-jboss-eap.md

cf https://github.com/endoflife-date/endoflife.date/pull/2883#discussion_r1199804121

adriens committed:
remove LTS

cf https://github.com/endoflife-date/endoflife.date/pull/2883#discussion_r1199806643

adriens committed:
Update redhat-jboss-eap.md

https://github.com/endoflife-date/endoflife.date/pull/2883#discussion_r1199806764

adriens committed:
Summarizing ELS notes

cf https://github.com/endoflife-date/endoflife.date/pull/2883#discussion_r1199808389

adriens committed:
Update redhat-jboss-eap.md

https://github.com/endoflife-date/endoflife.date/pull/2883#discussion_r1199805278
https://access.redhat.com/articles/6999113

adriens committed:
Update redhat-jboss-eap.md

cf https://github.com/endoflife-date/endoflife.date/pull/2883#discussion_r1199806037

adriens committed:
Update redhat-jboss-eap.md

cf https://github.com/endoflife-date/endoflife.date/pull/2883#discussion_r1199806805

adriens committed:
Update redhat-jboss-eap.md

cf https://github.com/endoflife-date/endoflife.date/pull/2883#discussion_r1199806973
